api/services/sandbox/v1/sandbox.proto

```diff
@@ -120,6 +120,9 @@ message ControllerStartResponse {
 	uint32 pid = 2;
 	google.protobuf.Timestamp created_at = 3;
 	map<string, string> labels = 4;
+	string address = 5;
+	uint32 version = 6;
+	string protocol = 7;
```

Comment:
Can we pack protocol and address in one Url like field to carry less new fields around?

Other then that, looks good.

Reply:
The address is url like 'unix://path/to/uds', protocol may only be added into the query fields? `unix://<uds-path>/<version>?protocol=<protocol>`

Reply:
Can it be `protocol://address` ?

Reply:
it is because the address already has a theme field like `unix:://` or `vsock://` if we add the protocol before the address then it will be like `ttrpc://unix://tmp/test.sock`.

Reply:
possibly `ttrpc+vsock://path`?

Reply:
done, the address returned from sandbox controller should be in the form like ttrpc+vsock://<cid>:<port> or grpc+unix://path